pytest_elk_reporter.py

```diff
@@ -196,6 +196,7 @@ def pytest_runtest_logreport(self, report):
     def report_test(self, item_report, outcome, old_report=None):
         self.stats[outcome] += 1
         test_data = dict(
+            item_report.user_properties,
```

Comment:
should that be something like `**item_report.user_properties`, i.e. isn't `item_report.user_properties` a dict ?